.gitmodules

```diff
@@ -174,3 +174,6 @@
 [submodule "community-build/community-projects/scalatestplus-junit"]
 	path = community-build/community-projects/scalatestplus-junit
 	url = https://github.com/dotty-staging/scalatestplus-junit.git
+[submodule "community-build/community-projects/perspective"]
+	path = community-build/community-projects/perspective
+	url = https://github.com/Katrix/perspective.git
```

Comment:
Is it ok to use the original repo directly instead of a fork? If we don't have permissions to fully manage this repo, we might not be able to fix our tests in case some changes are pushed there without all tests passing for scala 3 or with flaky tests. @smarter ?

Reply:
Our current policy is to always make a  fork in dotty-staging indeed.

Reply:
@smarter I could create the fork but it looks like I don't have the right permissions for that

Reply:
You should have access now.

Reply:
@Katrix you can now use
```
https://github.com/dotty-staging/perspective.git
```
instead